test-server/Sources/App/routes.swift

```diff
@@ -5,18 +5,13 @@ func routes(_ app: Application) throws {
         return "It works!"
     }
 
-    app.get("hello") { request -> String in
-        let tracestate = request.headers["baggage"]
-        if let sentryTraceHeader = tracestate.first {
-            // We just validate if the trace baggage header is there.
-            // The proper format of the trace header is covered
-            // with unit tests.
-            if sentryTraceHeader.starts(with: "sentry-") {
-                return "Hello, world! Trace header added."
-            }
+    app.get("echo-baggage-header") { request -> String in
+        let baggage = request.headers["baggage"]
+        if let sentryTraceHeader = baggage.first {
+          return sentryTraceHeader
```

Comment:
`Hello, world! Trace header added.` seems way less useful than echoing the actual header, so we can test against that. I've modified the existing test that hits the test server, but I think we don't need the test server at all any more, see comment above.

Reply:
I agree that echoing information is way more useful.